randovania/games/fusion/logic_database/Sector 1 SRX.txt

```diff
@@ -1432,12 +1440,10 @@ Extra - room_id: [40]
               # Vanilla Method - Shinespark from Hornoad Housing -> Access
               Speed Booster and Disabled Door Lock Rando and Disabled Entrance Rando and Can Kill Eye Door
               Any of the following:
-                  # Pre-Fight, with Hi-Jump
-                  Hi-Jump and Shinespark Tricks (Intermediate)
-                  # After Fight, Trickless
-                  Hi-Jump and After Boss Charge Core-X Defeated
-                  # No Hi-Jump
-                  Shinespark Tricks (Advanced)
+                  # Trickless: https://youtu.be/Z6_miH4qoQY
+                  Hi-Jump
```

Comment:
you considerably changed logic here. please revert this back to what it was:
HJ + boss killed => trickless
HJ + boss alive => intermediate (softlock potential)
HJless => advanced (softlock potential + less lenient)


Alternatively I would also accept
HJ + boss alive =>  intermediate (softlock potential)
HJless + boss killed => intermediate (less lenient)
HJless + boss alive => advanced (softlock potential + less lenient)

Reply:
Reverted the logic and added the videos for the existing logic.